core/git-utils/index.js

```diff
@@ -209,6 +209,41 @@ function hasCommit(opts) {
   return retVal;
 }
 
+function fetchGitRemote(opts) {
+  log.silly("fetchGitRemote");
+  ChildProcessUtilities.execSync("git", ["remote", "update"], opts);
+}
+
+function isBehindUpstream(gitRemote, opts) {
+  log.silly("isBehindUpstream");
+
+  const status = exports.aheadBehindCount(gitRemote, opts);
+  const behind = status.behind >= 1;
+
+  log.verbose("isBehindUpstream", behind);
+  return behind;
+}
+
+function aheadBehindCount(gitRemote, opts) {
+  const branchName = exports.getCurrentBranch(opts);
+  const branchComparator = `${gitRemote}/${branchName}...${branchName}`;
+  const rawAheadBehind = ChildProcessUtilities.execSync(
+    "git",
+    ["rev-list", "--left-right", "--count", branchComparator],
+    opts
+  );
+
+  const aheadBehind = rawAheadBehind.split("");
```

Comment:
The `--count` flag always separates the values with a tab, we should split on `"\t"` here (`execa` already trims the `stdout` result). Unless you're accounting for other cases I'm not seeing?